modules/concept-docs/pages/durability-replication-failure-considerations.adoc

```diff
@@ -1,8 +1,8 @@
 = Durability & Failure
 :description: Data durability refers to the fault tolerance and persistence of data in the face of software or hardware failure.
 :page-topic-type: concept
-// :page-aliases: ROOT:failure-considerations,ROOT:durability,ROOT:enhanced-durability
-// :page-aliases: 7.0@server:sdk:durability.adoc
+:page-aliases: ROOT:failure-considerations,ROOT:durability,ROOT:enhanced-durability
+//:page-aliases: 7.0@server:sdk:durability.adoc
```

Comment:
Not sure if `7.0@server:sdk:durability.adoc` should be removed as it's a server page...? @RichardSmedley

Reply:
I _think_ not only should it possibly still be there,
as it is in 3.0,
but there needs to be one the same, but with developer-guide in place of :sdk:
for content present in 4.x docs.
Not essential, though.